plugins/source/github/resources/plugin/plugin.go

```diff
@@ -0,0 +1,30 @@
+package plugin
+
+import (
+	"github.com/cloudquery/cloudquery/plugins/source/github/client"
+	"github.com/cloudquery/plugin-sdk/plugins"
+)
+
+const (
+	exampleConfig = `
+		access_token: <Access Token HERE>
```

Comment:
This should be with no tab - otherwise `gen` wont work.

Reply:
cc @candiduslynx

Reply:
done